### PR TITLE
h2 2.0.202

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by SMillerDev — CHANGES_REQUESTED** [review]
Please remove the fixes in the install block

File: Formula/h2.rb [hunk 1]
```diff
@@ -21,14 +21,6 @@ def install
...
-    # As of 1.4.190, the script contains \r\n line endings,
-    # causing it to fail on macOS. This is a workaround until
-    # upstream publishes a fix.
-    #
-    # https://github.com/h2database/h2database/issues/218
-    h2_script = File.read("bin/h2.sh").gsub("\r\n", "\n")
-    File.open("bin/h2.sh", "w") { |f| f.write h2_script }
-
     # Fix the permissions on the script
     chmod 0755, "bin/h2.sh"
```
Comment (SMillerDev) [inline Formula/h2.rb:33]: This one too

**Review by SMillerDev — CHANGES_REQUESTED** [review]

File: Formula/h2.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/h2.rb:33]: I  checked the permission of the file, looks like we still need 0755 for it?

Comment (chenrui333) [reply Formula/h2.rb:33]: Anyway, let me see if it can be removed or not.

Comment (SMillerDev) [reply Formula/h2.rb:33]: If it does that should be reported upstream.

Comment (chenrui333) [reply Formula/h2.rb:33]: sure, I will raise this up to the upstream.

Comment (chenrui333) [reply Formula/h2.rb:33]: https://github.com/h2database/h2database/issues/3254

**Comment by chenrui333** [comment]
Good for the merge?

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**